easybuild/easyconfigs/s/SMRT-Link/SMRT-Link-12.0.0.177059-cli-tools-only.eb

```
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Binary'
+
+name = "SMRT-Link"
+version = "12.0.0.177059"
+versionsuffix = '-cli-tools-only'
+
+homepage = 'https://www.pacb.com/support/software-downloads/'
+description = """PacBio’s open-source SMRT Analysis software suite is designed for use with Single Molecule, 
+ Real-Time (SMRT) Sequencing data. You can analyze, visualize, and manage your data through an intuitive GUI 
+ or command-line interface. You can also integrate SMRT Analysis in your existing data workflow through 
+ the extensive set of APIs provided"""
+
+toolchain = SYSTEM
+
+# The download is a password protected zip file
+# Download it from https://www.pacb.com/support/software-downloads/ and uncompress
+# the .run file to your sources folder
```

We can use `download_instructions` here. This will print the message when the file is not found in the source folder.

```suggestion
download_instructions = """The download is a password protected zip file
Download it from https://www.pacb.com/support/software-downloads/ and uncompress
the .run file to your sources folder"""
```